public/app/features/explore/Wrapper.tsx

```diff
@@ -25,7 +25,7 @@ export class Wrapper extends Component<WrapperProps> {
     return (
       <div className="page-scrollbar-wrapper">
         <CustomScrollbar autoHeightMin={'100%'} autoHeightMax={''} className="custom-scrollbar--page">
-          <div style={{ height: '100%' }} className="explore-wrapper">
```

Comment:
Was needed before when we used iframe for the ui

Reply:
👍